PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java

```diff
@@ -25,9 +25,13 @@ public static void main(String[] args) {
         Scanner keyboard = new Scanner(System.in);
         System.out.println("Welcome to your personal Twitter!");
         System.out.println("What's your name, tweeter?");
+        System.out.println(getCurrentTimeStamp());
```

Comment:
Good job!! but n you should move the getCurrentTimeStamp() line to the NewTweet method.